RELEASE_NOTES.md

```diff
@@ -0,0 +1,22 @@
+Release 3.1.1 of the Amazon Kinesis Video C++ Producer SDK
+
+Platforms tested on
+
+Linux
+Windows
+MacOS
+
+Release tagged at: 2d653a29451bce60741a4aeebb5825518345d25f
+
+Whats new:
+
+- Introduced a new kvssink property `disable-buffer-clipping` to cater to  some src/mux elements that produce non-standard segment start times
+- Added unit test for resetting and re-creating the stream
```

Comment:
Two more items perhaps to mention

* Overloaded constructors for supplying the caching mode
* C Producer: Resetting stream will not purge the API caching entries